components/colorpicker/overview.md

```diff
@@ -10,92 +10,100 @@ position: 0
+## ColorPicker Parameters
 
-### Custom Icon
+The Blazor ColorPicker provides various parameters to configure the component. For more configuration options, see the [public Telerik UI for Blazor ColorPicker API](/blazor-ui/api/Telerik.Blazor.Components.TelerikColorPicker).
 
-The ColorPicker provides attributes to render an icon inside the [main component button](#interface). This icon can be used to visually distinguish different ColorPickers on the page. In such cases, the selected color is displayed below the icon.
+The ColorPicker tag exposes the following features through its attributes:
 
-Use one attribute at a time:
+| Parameter | Type and Default Value | Description |
+| --- | --- | --- |
+| `Value` | `string` | Sets the ColorPicker value in a few [different color formats](#supported-value-formats). Supports two-way binding. |
+| `ValueFormat` | `ColorFormat` | Sets the color format, which the component will return in the application code. By default, the `ValueFormat` is set to `Rgb`, but it can be changed to `Hex`. |
+| `ColorPickerViews` | `RenderFragment` | A nested container to list the [ColorPicker views]({%slug colorpicker-views%}). All views are enabled by default and the user can switch between them with the buttons. Each view tag has its own configuration attributes. |
+| `View` | `ColorPickerView` | Sets the default selected [view]({%slug colorpicker-views%}) (`ColorPickerView.Gradient` by default). Supports two-way binding. |
+| `ShowPreview` | `bool` | Toggles the [current color box and the color preview box](#interface) in the popup (`true` by default). |
+| `Class` | `string` | Renders a custom CSS class to the `span.k-colorpicker` element. |
+| `Enabled` | `bool` | Determines if the user can open the popup and change the value (`true` by default).
+| `ShowButtons` | `bool` | Sets the visibility of the **Apply** and **Cancel** buttons (`true` by default).
+| `ShowClearButton` | `bool` | Sets the visibility of the **Clear** button.
 
-* `Icon` - `object` - specifies a [built-in Font or SVG icon]({%slug general-information/font-icons%}).
+### Styling and Appearance
 
+The following parameters enable you to customize the appearance of the Blazor ColorPicker:
 
-## Supported Value Formats
+| Parameter | Type | Description |
+| --- | --- | --- |
+| `Size` | `Telerik.Blazor.ThemeConstants.ColorPicker.Size` | Adjusts the size of the ColorPicker. |
+| `Rounded` | `Telerik.Blazor.ThemeConstants.ColorPicker.Rounded` | Affects the `border-radius` of the ColorPicker. |
+| `FillMode` | `Telerik.Blazor.ThemeConstants.ColorPicker.FillMode` | Controls how the ColorPicker is filled. |
 
-The ColorPicker accepts values by the application code in the following formats:
+For more information on customizing the ColorPicker styling, see the article about [setting the appearance of the ColorPicker]({%slug colorpicker-appearance%}).
 
-@[template](/_contentTemplates/common/coloreditors.md#value-formats)
+### Custom Icons
```

Comment:
This section can (and should) be replaced with a simple row in the Parameters table above.